modules/c-blosc2/2.12.0.bcr.2/presubmit.yml

```diff
@@ -0,0 +1,23 @@
+matrix:
+  platform:
+    - debian10
+    - debian11
+    - macos
+    - macos_arm64
+    - ubuntu2004
+    - ubuntu2204
+    - windows
+  bazel: [6.x, 7.x, rolling]
+tasks:
+  verify_targets:
+    name: Verify build targets
+    platform: ${{ platform }}
+    bazel: ${{ bazel }}
+    build_targets:
+      - "@c-blosc2//:c-blosc2"
+  run_tests:
+    name: Run tests
+    platform: ${{ platform }}
+    bazel: ${{ bazel }}
+    test_targets:
+      - "//..."
```

Comment:
This has to be `@c-blosc2//...` or you have to use `bcr_test_module` with `module_path = "."`, otherwise it's running under a dummy module that depends on `c-blosc2`
See https://github.com/bazelbuild/bazel-central-registry/blob/main/docs/README.md#test-module

Reply:
Thanks for the hint!